### PR TITLE
chore: Add zookeeper 3.8.4

### PR DESCRIPTION
**Pull request by NickLarsenNZ**
# Description

Bump zookeeper to 3.8.4

## Definition of Done Checklist

- Not all of these items are applicable to all PRs, the author should update this template to only leave the boxes in that are relevant
- Please make sure all these things are done and tick the boxes
 
```[tasklist]
- [ ] Changes are OpenShift compatible
- [ ] All added packages (via microdnf or otherwise) have a comment on why they are added
- [ ] Things not downloaded from Red Hat repositories should be mirrored in the Stackable repository and downloaded from there
- [ ] All packages should have (if available) signatures/hashes verified
- [ ] Does your change affect an SBOM? Make sure to update all SBOMs
```

### CODE REVIEW

File: conf.py [hunk 1]
```diff
@@ -390,7 +390,7 @@
         "name": "zookeeper",
         "versions": [
             {
-                "product": "3.8.3",
+                "product": "3.8.4",
```
Comment (sbernauer) [inline conf.py:393]: We need 3.8.3 **and** 3.8.4 please (have a look at the Supported versions sheet)

**Review by sbernauer — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -58,6 +58,7 @@ All notable changes to this project will be documented in this file.
 - hdfs: bump topology-provider to `0.3.0` ([#579]).
 - ubi8-rust-builder: bump rust toolchain to `1.76.0` ([#584]).
 - opa: bump bundle builder to version 1.1.1 ([#585]).
+- zookeeper: bump to `3.8.4` ([#591]).
```
Comment (sbernauer) [inline CHANGELOG.md:61]: Nit: Normally we add new versions to "Added" above

`- zookeeper: Add version `3.8.4` ([#591]).`

Comment (NickLarsenNZ) [reply CHANGELOG.md]: Resolved in 0fcf8392f636992d7e27109ac0a5ca3e7e5d5347

**Review by sbernauer — APPROVED** [review]
Thanks!

**Pull request merged**